docs/interactions/Application_Commands.md

```diff
@@ -82,6 +87,17 @@ All options have names, and an option can either be a parameter and input value-
 | value?   | [application command option type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-type)                                    | the value of the pair                                                                                                                    |
 | options? | array of [application command interaction data option](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/interaction-object-application-command-interaction-data-option-structure) | present if this option is a group or subcommand                                                                                          |
 
+
+###### Application Command Autocomplete Interaction Data Option Structure
+
+| Field    | Type    | Description                                                                                                                                    |
+| -------- |-------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| name     | string  | the name of the parameter                                                                                                                      |
+| type     | integer | value of [application command option type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-type) |
+| value?   | string  | value of the command option                                                                                                                    |
```

Comment:
Is this value *always* going to be a string? I still need to test but I think it may be possible that it is not

Reply:
It was indicated as string in devsneks notation.
But yeah, please check on it

Reply:
Tbh, quite interesting.
I'm gonna take a look into it later myself.

Reply:
Talked with mason (selfnote: https://discord.com/channels/@me/872555058888314940/890720127014805514)
>It should be able to be a type of whatever option types support choices

Reply:
yeah, that's what it seemed like to me. 

Although I'd like it to accept a string always so you can search int choices by name instead of value